package.json

```diff
@@ -19,15 +19,15 @@
     "test:watch": "vitest",
     "test:size": "npm i && size-limit",
     "test:coverage": "vitest run --coverage",
-    "test:browser": "playwright test /tests",
+    "test:browser": "playwright test",
     "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --compiler-warnings \"css-unused-selector:ignore,unused-export-let:error\"",
     "lint": "eslint . --fix",
     "format": "prettier --write .",
     "host": "npm run all && npm run preview",
     "all": "npm run lint && npm run format && npm run check && npm run build && npm t"
   },
   "devDependencies": {
-    "@playwright/test": "^1.25.0",
+    "@playwright/test": "^1.28.0-alpha-oct-12-2022",
```

Comment:
Is there a specific reason for using this version? Using alpha seems unnecessary

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/commit/829a50bfa2b0cdad02a5d70ecfb0814d7e20da26>.